rotkehlchen/chain/ethereum/modules/safe/constants.py

```diff
@@ -0,0 +1,7 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+CPT_SAFE: Final = 'safe'
```

Comment:
we already have `CPT_SAFE_MULTISIG`. Does it make sense to keep only `safe` and not have multiple safe things?

Reply:
It's not the same protocol. One is the multisig. The other is the token. Not sure if it makes sense